.github/workflows/ci.yaml

```diff
@@ -106,14 +112,19 @@ jobs:
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
       - name: Run cargo build with devnet-prealloc feature
-        run: cargo build --features devnet-prealloc --workspace --all --tests --benches
-        
+        run: cargo build --release --features devnet-prealloc --workspace --all --tests --benches
+
       - name: Run cargo test regular features
-        run: cargo test --workspace
+        run: cargo nextest run --release --workspace
+
+      - name: Run cargo doc tests with features=no-asm
+        run: cargo test --doc --release --workspace
 
       - name: Run cargo test on kaspa-hashes without asm
-        run: cargo test -p kaspa-hashes --features=no-asm --benches
+        run: cargo nextest run --release -p kaspa-hashes --features=no-asm --benches
 
+      - name: Run cargo doc tests with features=no-asm
+        run: cargo test --doc --release --features=no-asm
```

Comment:
IMHO this one can be narrowed down to
`cargo test --doc --release -p kaspa-hashes --features=no-asm --benches`

Reply:
It can't because `--benches` and `-p kaspa-hashes` is another target:

```
rusty-kaspa git:(rpc-integration-tests) ✗ cargo test --doc --release -p kaspa-hashes --features=no-asm --benches
error: Can't mix --doc with other target selecting options
```

Reply:
Removing `--benches` and keeping `-p kaspa-hashes` works so doing that now.